community-call/20221208.md

```diff
@@ -0,0 +1,103 @@
+# SCS Tech Weekly 2022-12-01
+
+## Participants  use of github handle is preferred
+* @itrich
+* @o-otte
+* @garloff
+* @alexander-diab
+* @manuela-urban
+* @tibeer
+* @reqa
+* @JuanPTM
+* @ppkuschy
+* @maxwolfs
+* @fungi
+
+## Team Updates
+
+### Team IaaS (Sprint review and planning)
+* Test setup with multiple Keycloaks
+* Validation limited to ceph clusters deployed by OSISM
+* image-manager documentation proceeding
+* image-crawler can crawl three image at the moment. Needs to be integrated with image-manager
+* SOS-report functionality on hold at the moment
+* Benchmarks to determine overhead of disk encryption in ceph
+* Wiping data from local disks needs work (currently lvremove is insufficient!)
+* Cloud-in-a-box node image works well
+    
+Minutes: https://github.com/SovereignCloudStack/minutes/blob/main/iaas/20221207.md
+
+### Team Container (Sprint review and planning)
+* Local storage ADR was merged
+* Test cases for loadbalancer beahvior
+* Tests should be added to a test collection
+* Needs some research on appropriate test suites
+* Next:  ADR on loadbalancer behavior and what needs to be changed in the reference implementation
+* PRs open in k8s-cluster-api implementation on the latest k8s patch updates - review wanted
+* ADR on k8s update policy needed
+* Next weeks: Additional storage classes discussion on 2022-12-19
+
+Minutes: https://github.com/SovereignCloudStack/minutes/blob/main/container/20221205.md
+
+**No team meeting on 2022-12-12!**
+
+### Team Ops, IAM (Sprint review and planning)
+
+* Review of work happening on device-authorization grant by @reqa
+* PoC works, yeah!
```

Comment:
This and the next one: Additional indentation (they are subitems to device auth grant)